client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/SubmitCommand.kt

```diff
@@ -33,13 +36,17 @@ internal class SubmitAsyncCommand(
     private val attached by option("-a", "--attached", help = ATTACHED_HELP)
     private val preferredSources by option("-ps", "--preferredSources", help = PREFERRED_SOURCES)
     private val storageMode by option("-sm", "--storageMode", help = STORAGE_MODE).default(FIRE.value)
+    private val timeout by option("-tm", "--timeout", help = TIMEOUT).int().default(0)
```

Comment:
I think agreetment here is wait indefinitly if `wait = true` if a user want to stop waiting the can simple close application.

Reply:
Right